internal/datastore/crdb/readwrite.go

```diff
@@ -267,8 +268,29 @@ func (rwt *crdbReadWriteTXN) WriteNamespaces(newConfigs ...*core.NamespaceDefini
 	return nil
 }
 
-func (rwt *crdbReadWriteTXN) DeleteNamespace(nsName string) error {
-	ctx := datastore.SeparateContextWithTracing(rwt.ctx)
+func (rwt *crdbReadWriteTXN) DeleteNamespaces(nsNames ...string) error {
+	ctx, span := tracer.Start(
+		datastore.SeparateContextWithTracing(rwt.ctx),
+		"DeleteNamespaces",
+		trace.WithAttributes(attribute.StringSlice("names", nsNames)),
+	)
+	defer span.End()
+
+	for _, nsName := range nsNames {
```

Comment:
We want to do this in a single statement instead of looping over, but I'm OK leaving it that way right now since it's the case already, and this PR adjusts the interface to accept multiple namespaces. We should open a follow up issue to address this 👍🏻